src/utilities/utilities.hpp

```diff
@@ -47,6 +62,12 @@ constexpr auto kUnknownError = -999;
 
 // =================================================================================================
 
+constexpr auto tStrategy = "strategy";
+constexpr auto psoSwarmSize = "psoSwarmSize";
+constexpr auto  psoInfG = "psoInfG";
+constexpr auto  psoInfL = "psoInfL";
+constexpr auto  psoInfR = "psoInfR";
+
```

Comment:
Can you move these a bit lower under the section with the tuning-specific arguments and also follow their format: `kArgCamelCaseName` and the string itself as `lower_case_with_underscores`? But actually these are quite specific for this particular case of tuning, so perhaps we should move them out of the arguments structure and add them separately just for the tuners. That is a bit nicer IMHO, but it could have been done for some other existing cases as well...